lib/client.js

```diff
@@ -19,8 +17,8 @@ function Client () {
         public: 'https://beam.pro'
     };
 
-    for (var key in Services) {
-        this[key] = new Services[key](this);
+    for (var keys = Object.keys(Services), i = 0, length = keys.length; i < length; i++) {
+        this[keys[i]] = new Services[keys[i]](this);
     }
```

Comment:
If you want to avoid prototype properties, do `Object.keys(Services).forEach(function (key) {`